text/0000-reseeding-rand.md

```diff
@@ -0,0 +1,657 @@
+- Start Date: 2013-01-23
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Stabilise the `std::rand` module by focusing it on a smaller set of
+functionality, moving "fancy" functionality to a crates.io crate.
```

Comment:
s/crate/crates/?

ISAAC will get its own crate and distributions would get another.